css/style.css

```diff
@@ -16,4 +16,243 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* Header */
 
+ header {
+     display: flex;
+     justify-content: space-between;
+     align-items: center;
+     padding: 1rem 3rem;
+ }
+
+ .svg-logo {
+     height: 2.5rem;
+ }
+
+ @media only screen and (max-width: 768px) {
```

Comment:
This really is the extra mile! I'm not reviewing the media queries fully as it's apparently not intended to be part of the exercise but it looks great on my machine 🎉

Reply:
Thanks Sam.